js/index.js

```diff
@@ -82,11 +80,12 @@ function passwordScore(pass) {
     }
     return {score: score, error: error};
 }
-$('.signup-form').find('#password').keyup(function(ele) {
-	var scoreData = passwordScore($(this).val()),
+
+function passwordFieldKeyUp(t) {
```

Comment:
Also, rather than using "ele", lets use variable names that represent what is being processed. Avoid variable names like 'ele', 't' etc.